.circleci/workflows.yml

```diff
@@ -232,6 +232,138 @@ jobs:
             - ./gravitee-am-management-api-standalone-*.zip
             - ./gravitee-am-webui-*.zip
 
+  run_jest_tests_mongo:
+    docker:
+      - image: cimg/openjdk:17.0.1-node
+      - image: mongo:4.2
+    resource_class: small
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout
+      - run:
+          name: install test module dependencies
+          command: npm --prefix gravitee-am-test i
+      - run:
+          name: Wait for Mongo to be up and running
+          command: timeout 15s bash -c 'until nc -vz localhost 27017; do sleep 2; done'
+      - run:
+          name: Run Jest tests
+          command: |
+            # Prepare AM
+            unzip /tmp/workspace/gravitee-am-gateway-standalone-*.zip
+            unzip /tmp/workspace/gravitee-am-management-api-standalone-*.zip
+            export AM_VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
+            curl https://download.gravitee.io/plugins/services/gravitee-service-geoip-1.2.0.zip -o gravitee-am-gateway-standalone-${AM_VERSION}/plugins/gravitee-service-geoip-1.2.0.zip
+            curl https://repo.maven.apache.org/maven2/io/gravitee/am/resource/gravitee-am-resource-mfa-mock/3.12.1/gravitee-am-resource-mfa-mock-3.12.1.zip -o gravitee-am-gateway-standalone-${AM_VERSION}/plugins/gravitee-am-resource-mfa-mock-3.12.1.zip
+            cp gravitee-am-gateway-standalone-${AM_VERSION}/plugins/gravitee-am-resource-mfa-mock-3.12.1.zip gravitee-am-management-api-standalone-${AM_VERSION}/plugins/
+            # Start AM Management
+            gravitee-am-management-api-standalone-${AM_VERSION}/bin/gravitee &
+            # Start AM Gateway
+            gravitee-am-gateway-standalone-${AM_VERSION}/bin/gravitee &
+            # build & start CIBA Delegated Service
+            mvn -P cicd -f gravitee-am-ciba-delegated-service/pom.xml package
+            java -jar gravitee-am-ciba-delegated-service/target/gravitee-am-ciba-delegated-service-${AM_VERSION}.jar &
+            sleep 30
+            # Wait for AM to be up and running
+            timeout 30s bash -c 'until nc -vz localhost 8093; do sleep 2; done'
+            timeout 30s bash -c 'until nc -vz localhost 8092; do sleep 2; done'
+            sleep 30
+            # Run Jest tests
+            npm --prefix gravitee-am-test run ci
+      - slack/notify:
+          branch_pattern: ".+"
+          event: fail
+          mentions: "@am_team"
+          template: basic_fail_1
+      - slack/notify:
+          branch_pattern: ".+"
+          event: pass
+          mentions: "@am_team"
```

Comment:
Maybe we should remove the @am_team notification in case of success, wdyt?
Having the success message into the chanel should be enough. We want to be notified only for failure. (if you are agree, can you also do it for postman executions? )

Reply:
Completely agree 👍